web-client/src/presenter/actions/TrialSession/setPrintPaperDoneUrlAction.test.js

```diff
@@ -0,0 +1,35 @@
+import { presenter } from '../../presenter-mock';
+import { runAction } from 'cerebral/test';
+import { setPrintPaperDoneUrlAction } from './setPrintPaperDoneUrlAction';
+
+describe('setPrintPaperDoneUrlAction', () => {
+  it('sets the printPaperDoneUrl to /trial-sessions when trialSessionId is not on props', async () => {
+    const result = await runAction(setPrintPaperDoneUrlAction, {
+      modules: {
+        presenter,
+      },
+      props: {},
+      state: {
+        printPaperDoneUrl: null,
+      },
+    });
+
+    expect(result.state.printPaperDoneUrl).toEqual('/trial-sessions');
+  });
+
+  it('sets the printPaperDoneUrl to /trial-session-detail/abc when props.trialSessionId is set to abc', async () => {
+    const result = await runAction(setPrintPaperDoneUrlAction, {
+      modules: {
+        presenter,
+      },
+      props: {
+        trialSessionId: 'abc',
+      },
+      state: {
+        printPaperDoneUrl: null,
+      },
+    });
+
+    expect(result.state.printPaperDoneUrl).toEqual('/trial-session-detail/abc');
```

Comment:
Nit: extract `abc` into a variable that is used in running the test and in the expect.